integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CommonLoadUtils.scala

```diff
@@ -1208,3 +1228,46 @@ object CommonLoadUtils {
     segmentMetaDataInfo
   }
 }
+
+/**
+ *  It unwraps partition from DataLoadPartitionWrap
+ */
+class DataLoadCoalescedUnwrapRDD[T: ClassTag](@transient var prev: RDD[T])
```

Comment:
please add comments for this wrapper class for better understanding

Reply:
Done